build-tools/build-image/cross/Dockerfile

```diff
@@ -51,14 +53,13 @@ RUN mkdir -p /usr/local/src/protobuf \
 
 # Use dynamic cgo linking for architectures other than amd64 for the server platforms
 # More info here: https://wiki.debian.org/CrossToolchains
-RUN echo "deb http://emdebian.org/tools/debian/ jessie main" > /etc/apt/sources.list.d/cgocrosscompiling.list \
-  && curl -s http://emdebian.org/tools/debian/emdebian-toolchain-archive.key | apt-key add - \
-  && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do dpkg --add-architecture ${platform}; done \
+RUN echo "deb http://cz.archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/sources.list.d/cgocrosscompiling.list \
```

Comment:
@luxas @ixdy  I have been scratching my head to resolve this problem. Actually with the existing (jessie main) repository, there is no crossbuild-essential-s390x available. For cross compilation we need s390x-linux-gnu-gcc binary . I tried your suggested approach keeping the "jessie main" for existing supported architectures and introducing new repo for only for s390x but this solution is not working. Keeping two repositories cause dependencies and version issues error for the packages (like libc and glibc) which are used by s390x and existing architecture. I think there can be 2 ways to address this problem.
1. We can upgrade the repository to xenial as I did in the PR. I think this approach is quite feasible because having the latest stable repository  will not break anything also it will provide higher versions of packages and down the line we will have to upgrade it to latest repositories. 
2. For s390x only I can include the steps for cross compiling gcc from the ource so that it would create s390x-linux-gnu-gcc binary and which will be used for compiling Kubernetes on s390x. Once we upgrade or enable the proper repository we would remove this this step. 

I will need your inputs to decide a better solution.